cmd/k8s-device-plugin/main.go

```diff
@@ -129,14 +130,39 @@ func (p *Plugin) ListAndWatch(e *pluginapi.Empty, s pluginapi.DevicePlugin_ListA
 
 	devs := make([]*pluginapi.Device, len(p.AMDGPUs))
 
+	var hw hwloc.Hwloc
+	hw.Init()
+
 	i := 0
 	for id := range p.AMDGPUs {
 		devs[i] = &pluginapi.Device{
 			ID:     id,
 			Health: pluginapi.Healthy,
 		}
+
+		numas, err := hw.GetNUMANodes(id)
+		glog.Infof("Watching GPU with bus ID: %s NUMA Node: %+v", id, numas)
+
+		if err == nil && len(numas) > 0 {
```

Comment:
Idiomatic go would try and handle the error case and "continue" rather than have an "else" clause.
Obviously in order to make this work you need to move the i++ to before this if, and start i at -1.

Perhaps split this into 2 tests?
i++
if err != nil { glog.Error(err) ; continue; }
if len(numas) == 0 { continue ; }
and then outdent the rest of the code.